content/assignments/techjobs-console/_index.md

```diff
@@ -0,0 +1,88 @@
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Introduction
+
+Congratulations! Based on your hard work and strong coding skills, you have
+been brought on as an apprentice to the LaunchCode Tech Team. You will be
+paired with a mentor to help you get comfortable and continue learning.
+
+The Company Team at LaunchCode works with employer partners to match qualified
+programmers with apprenticeships. They asked for a new tool to be built to
+help them easily manage data for currently available jobs. Over the next few
+weeks, you will help them build this application alongside mentors from the
+Tech Team.
+
+This first project will be a simple proof-of-concept prototype. It won't be
+pretty or have lots of features, but it will give you a chance to work through
+some initial concepts and get feedback from LaunchCode staff.
+
+Your mentor on this project is Blake.
+
+{{< rawhtml >}}
+   <img src="pictures/LC-Blake.png" alt="LaunchCode Mentor Blake" width=20% />
+{{< /rawhtml >}}
+
+## Learning Objectives
+
+In this project, you will show that you can:
+
+1. Read and understand code written by others.
+2. Use core Java syntax (methods, variables, loops, conditionals).
+3. Utilize `ArrayList` and `HashMap` collection types.
+4. Work with console I/O via the `Scanner` class.
+5. Work with data types and arrays.
+
+## TechJobs (Console Edition)
+
+The app you must help design is a simple console (i.e. command-line) prototype
+of the new TechJobs app. It will allow LaunchCode staff to browse and search
+listings of open jobs by employer partners.
+
+The prototype process gives everybody a chance to work out some initial ideas
+without investing a ton of time into developing a finished product. Once
+everybody likes the prototype, the Tech Team will begin work toward a
+full-fledged application.
+
+## Your Assignment
+
+Blake created a console application and started to fill in some features. His
+code allows users to search job listings by one of several fields. It can also
+display lists of all of the values of a given field in the system (e.g. all
+employers, or all locations).
+
+Blake has now handed the task off to you. You must add a couple of features and
+then get feedback from the Company Team.
+
+After you work through the tasks Blake has laid out for you, tackle one
+or more of the [bonus missions](#tech-jobs-console-bonus-missions).
+
+## Getting Started
+
+In Canvas, **Graded Assignment #1: TechJobs Console** contains a GitHub Classroom assignment invitation link and then set up the project in IntelliJ. Refer back to the GitHub Classroom instructions from [assignment0](assignment0.md) for details. 
```

Comment:
Please update link

Reply:
Not sure if this link will be an issue once merged into main, as it seems to be hard coded instead of using relref, see above comment.